models/payment.py

```diff
@@ -147,6 +149,15 @@ def get_export_data(cls):
 
         return data
 
+    def is_refundable(self, override_refund_state_machine=False) -> bool:
```

Comment:
I'd suggest inverting the argument:

```suggestion
    def is_refundable(self, by_admin=False) -> bool:
```

The verbose reason: payment processing software often uses [state machine logic](https://en.wikipedia.org/wiki/Finite-state_machine) to co-ordinate the lifecycle of payments, purchases, refunds, disputes, etc; it'd be nice to avoid a naming conflict with that.

Reply:
I would rather not because `by_admin` doesn't really tell you anything about what it's being done whereas this tries to make it much clearer what the actual change is. 

there is definitely a risk of confusion with the other statemachines we have (as well as payment ones) but that's part of the reason for calling out the use of 'refund'

Reply:
Ok, yep.  Any chance of `ignore_event_refunds_state` or something similar instead?

Reply:
Thanks!